Dockerfile

```
@@ -0,0 +1,3 @@
+FROM python:3.12-slim
+
+COPY requirements.txt /requirements.txt
```

```suggestion
COPY requirements.txt /requirements.txt
COPY main.py /main.py

RUN ["main.py"]
```
the more the better

or even much more